tests/test_manifest_io.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import unittest
+import os
+from unittest import mock
+from chalicelib.utils import ManifestIO
+from requests.exceptions import HTTPError
+
+base_path = os.path.abspath(os.path.dirname(__file__))
+
+
+class TestManifestIO(unittest.TestCase):
+    # Use fixtures.
+    def setUp(self):
+        data = {
+            'participant': base_path + '/test_participant.tsv',
+            'sample': base_path + '/test_sample.tsv'
+        }
+        workspace = 'test_workspace'
+        namespace = 'test_namespace'
+        url = 'https://api.firecloud.org/api/workspaces'
+        statusCode = {
+            'status codes': {
+                'workspace exists': '',
+                'stood up workspace': '',
+                'participant upload': '',
+                'sample upload': ''
+            }
+        }
+        self.mani = ManifestIO(data=data, url=url,
+                               workspace=workspace, namespace=namespace,
+                               statusCode=statusCode)
+
+    def tearDown(self):
```

Comment:
The tearDown() method is executed after every test is run. It's intended for cleanup. You don't need the first two calls. I'm not even sure you need to set `self.main = None`.